packages/keystatic/src/app/dashboard/components.tsx

```diff
@@ -78,7 +78,7 @@ export const DashboardCard = (
                 textDecorationColor: tokenSchema.color.foreground.neutral,
 
                 '::before': {
-                  borderColor: tokenSchema.color.border.accent,
+                  borderColor: tokenSchema.color.border.emphasis,
```

Comment:
You want to keep `accentEmphasis` when [pressed](https://github.com/Thinkmill/keystatic/pull/522/files#diff-21caac9c730e2c2bbc5aad689480b464e974d474b2bf7f0b702d87f6a748dae7R86)?

Reply:
Yep, lgtm